src/main/res/values/strings.xml

```diff
@@ -17,25 +17,25 @@
   <string name="anti_feature_non_free_dependencies">Depends on non-free applications</string>
   <string name="anti_feature_non_free_network">Promotes non-free network services</string>
   <string name="anti_feature_tracking">Tracks or reports your activity</string>
-  <string name="anti_feature_upstream_non_free">Upstream source code is not free</string>
+  <string name="anti_feature_upstream_non_free">Upstream source code is not libre</string>
   <string name="anti_features">Anti-features</string>
   <string name="app_name">Foxy Droid</string>
   <string name="application">Application</string>
   <string name="application_not_found">Application not found</string>
-  <string name="authors_website">Author\'s website</string>
-  <string name="authors_email">Author\'s email</string>
+  <string name="authors_website">Author website</string>
+  <string name="authors_email">Author e-mail</string>
   <string name="available">Available</string>
   <string name="available_empty_summary">No applications available</string>
   <string name="bug_tracker">Bug tracker</string>
   <string name="cancel">Cancel</string>
   <string name="cant_edit_sync_description">Cannot edit repository since it\'s syncing right now.</string>
   <string name="changelog">Changelog</string>
   <string name="checking_repository">Checking repository</string>
-  <string name="compatible_with_only_format">Compatible with %s only</string>
-  <string name="confirm_action">Confirm action</string>
+  <string name="compatible_with_only_format">Only compatible with %s</string>
+  <string name="confirm_action">Confirm</string>
   <string name="connecting">Connecting</string>
   <string name="dark">Dark</string>
-  <string name="date_added">Date added</string>
+  <string name="date_added">Added</string>
```

Comment:
I also think the full "date added" should be used in this context.

Reply:
"Theme: Dark", "Added: <date>" has the same kind of logic to it.
What does "Date" add? Isn't it clear from looking at a <date> it was added on that day?

Reply:
The strings are ordered alphabetically, not by their usage in the program. "added" is not related to "dark". It's a name of sorting order in drop down menu.

Reply:
That is what I thought. Just for some savings and deeper consistency. Will change it back.

Reply:
Added ___. Is a better convention than "Date added ___"

Reply:
![device-2020-07-06-095535](https://user-images.githubusercontent.com/24494863/86564501-db43cb00-bf55-11ea-87e4-20314d9d86d4.png)